internal/gengapic/gengapic.go

```diff
@@ -42,8 +42,10 @@ import (
 
 const (
 	// protoc puts a dot in front of name, signaling that the name is fully qualified.
-	emptyType  = ".google.protobuf.Empty"
-	lroType    = ".google.longrunning.Operation"
+	emptyType = ".google.protobuf.Empty"
+	lroType   = ".google.longrunning.Operation"
+	// used when google.protobuf.Empty is the value of an annotation, which isn't resolved by protoc
+	emptyValue = "google.protobuf.Empty"
```

Comment:
Kind of a bummer to have two of the same import. Is there a way to tidy that?

Reply:
Yeah it is a bummer. Unfortunately, the preceding `.` has semantic meaning, it means protoc has identified it as a fully resolved name, which influences some code paths in the generator. However, when a user sets an annotation, which can be just a string value, to `"google.protobuf.Empty"`, this isn't mutated by protoc but influences different code paths in the generator. 

I will file a bug to look at refactoring it, but I'd prefer not to do it in this CL.

Reply:
Added a tracking bug comment.